kube-core/src/object.rs

```diff
@@ -68,11 +72,12 @@ impl<T: Clone> ObjectList<T> {
     /// # Example
     ///
     /// ```
-    /// use kube::api::{ObjectList, ListMeta};
+    /// use kube::api::{ListMeta, ObjectList, TypeMeta};
     ///
+    /// let types: TypeMeta = Default::default();
     /// let metadata: ListMeta = Default::default();
     /// let items = vec![1, 2, 3];
-    /// let mut objectlist = ObjectList { metadata, items };
+    /// let mut objectlist = ObjectList { types, metadata, items };
```

Comment:
This example is very awkward now as we are giving it technically illegal information and presenting it as it's fine. Let's hide the construction here from docs (because normally that comes from the api):
```suggestion
    /// # let mut objectlist = ObjectList { types, metadata, items };
```

and similar quote add for the lines above